app/experimenter/nimbus-ui/src/lib/config.test.ts

```diff
@@ -0,0 +1,123 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import config, {
+  getDefault,
+  readConfig,
+  decode,
+  reset,
+  update,
+} from './config';
+
+beforeEach(reset);
+
+describe('readConfig', () => {
+  it('merges the returned meta tag content value with the config', () => {
+    const data = {
+      version: '1.2.3'
+    }
+
+    const el = document.createElement('div');
+    el.dataset.config = encodeURIComponent(JSON.stringify(data))
+
+    readConfig(el);
+
+    expect(config.version).toBeDefined();
+    expect(config.version).toStrictEqual(data.version);
+  });
+});
+
+describe('decode', () => {
+  it('converts a JSON-stringified, URI-encoded string to a proper object', () => {
+    const data = {
+      decimated: 'dreams',
```

Comment:
😆

Reply:
oh shit I forgot I left MCR lyrics in here

Reply:
Hey, I leave 8675309 all over my code, live a little